json_test.go

```diff
@@ -188,14 +246,3 @@ func TestUnmarshalJSONString(t *testing.T) {
 	autopilot.Equals(t, data5, result5.AsMap())
 	autopilot.Assert(t, err != nil, "The JSON string of type bool should be unable to unmarshalled into a float32")
 }
-
-func TestNewJSONSchema(t *testing.T) {
-	res, err := ol.NewJSONSchema(validStringContainingJSON)
-	resVal := *res
-	autopilot.Ok(t, err)
-	autopilot.Equals(t, resVal["name"], "Thomas")
```

Comment:
Remove because [all these `equals` checks are done by reflect.DeepEqual anyways](https://stackoverflow.com/a/18211675), which is what autopilot.Equals is doing